index.bs

```diff
@@ -10454,86 +10462,62 @@ Constructors</h5>
-
-			* Nodes that transform their inputs, but which remain active
-				for a <a>tail-time</a> after their inputs are disconnected. In
-				this case, {{process()}} SHOULD return
-				`true` for some period of time after
-				<code>inputs</code> is found to contain zero channels. The
-				current time may be obtained from the global scope's
-				{{AudioWorkletGlobalScope/currentTime}} to
-				measure the start and end of this tail-time interval, or the
-				interval could be calculated dynamically depending on the
-				processor's internal state.
-
-			* Nodes that act as sources of output, typically with a
-				lifetime. Such nodes SHOULD return `true` from
-				{{process()}} until the point at which they are no
-				longer producing an output.
-
-			Note that the preceding definition implies that when no
-			return value is provided from an implementation of
-			{{process()}}, the effect is identical to returning
-			<code>false</code> (since the effective return value is the falsy
-			value {{undefined}}). This is a reasonable behavior for
-			any {{AudioWorkletProcessor}} that is active only when it has
-			active inputs.
-		</div>
-
-		<pre class=argumentdef for="AudioWorkletProcessor/process(inputs, outputs, parameters))">
-			inputs:
- 				The input audio buffer from the incoming connections provided by the user agent. It has type <code>sequence&lt;sequence&lt;Float32Array>></code>.<code class="nobreak">inputs[n][m]</code> is a {{Float32Array}} of audio samples for the \(m\)th channel of the \(n\)th input. While the number of inputs is fixed at construction, the number of channels can be changed dynamically based on [=computedNumberOfChannels=].
-
-				If there are no [=actively processing=] {{AudioNode}}s connected to the \(n\)th input of the {{AudioWorkletNode}} for the current render quantum, then the content of <code>inputs[n]</code> is an empty array, indicating that zero channels of input are available. This is the only circumstance under which the number of elements of <code>inputs[n]</code> can be zero.
-
-			outputs:
-				The output audio buffer that is to be consumed by the user agent. It has type <code>sequence&lt;sequence&lt;Float32Array>></code>.<code class="nobreak">outputs[n][m]</code> is a {{Float32Array}} object containing the audio samples for \(m\)th channel of \(n\)th output. Each of the {{Float32Array}}s are zero-filled. The number of channels in the output will match [=computedNumberOfChannels=] only when the node has a single output.
+The [=process()=] callback function is called
+synchronously by the audio <a>rendering thread</a> at
+every <a>render quantum</a>, if {{AudioWorkletNode}} is
+[=actively processing=].
 
-			parameters:
-				An [=ordered map=] of <var>name</var> → <var>parameterValues</var>. <code>parameters["<var>name</var>"]</code> returns <var>parameterValues</var>, which is a {{Float32Array}} with the automation values of the <var>name</var> {{AudioParam}}.
+This callback returns a {{boolean}}.
```

Comment:
The IDL shows the return type. Not sure if this is needed.

Reply:
Yes, this should be removed.